src/Features/CSharp/Portable/CSharpFeaturesResources.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Dieser Code wurde von einem Tool generiert.
```

Comment:
❗️ @tmeschter I have no idea what happened here, but I'm glad I expanded this designer file.

Reply:
why does this even get localized? that doesn't make any sense...